Test removing dependency wrapt <2.0.0

**Is your feature request related to a problem? Please describe.**
Btw @RhetTbull any reason to have wrapt dependency <2.0.0 ?

I'll try to remove it and test it. If tests pass I'll submit a PR.

**Describe the solution you'd like**
Consider wrapt version 2.2.2

**Describe alternatives you've considered**
N/A

**Additional context**
Possibly not an issue. Just keeping up with most recent versions of wrapt.

### Added also macOS-26 to test matrix: OK

### Local tests run on macOS 15.7.7
```
ProductName:		macOS
ProductVersion:		15.7.7
BuildVersion:		24G720
```


### Local pytest results: OK
```
=== 2736 passed, 251 skipped, 8 xfailed, 4 xpassed in 768.19s (0:12:48) ===
``` 

### run_scripts test 2: OK
```
====================================================================== 6 passed in 282.05s (0:04:42) ======================================================================

Test run completed!
```

### run_scripts test 3:  Getting some errors: will investigate.

<details>

<summary>
Several assertion errors:  False = compare_inspect_output(
</summary>

```
tests/test_cli_timewarp.py::test_date_added[2022-01-01-expected0] FAILED                                                     [ 91%]
tests/test_cli_timewarp.py::test_date_added[2022-01-01 01:02:03-expected1] FAILED                                            [ 93%]
tests/test_cli_timewarp.py::test_date_added_from_photo FAILED                                                                [ 95%]
tests/test_cli_timewarp.py::test_reset PASSED                                                                                [ 96%]
tests/test_cli_timewarp.py::test_album_query PASSED                                                                          [ 98%]
tests/test_cli_timewarp.py::test_year_query PASSED                                                                           [100%]

============================================================= FAILURES =============================================================
______________________________________________ test_date_added[2022-01-01-expected0] _______________________________________________

photoslib = <photoscript.PhotosLibrary object at 0x110797110>
suspend_capture = <tests.conftest.suspend_capture.<locals>.suspend_guard object at 0x11078cd70>, date_added = '2022-01-01'
expected = InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-19...te_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2022-01-01 00:00:00+0000')

    @pytest.mark.timewarp
    @pytest.mark.parametrize(
        "date_added,expected",
        TEST_DATA["date_added"]["data"],
    )
    def test_date_added(
        photoslib, suspend_capture, date_added: str, expected: InspectValuesDateAdded
    ):
        """Test --date-added"""
    
        runner = CliRunner()
        result = runner.invoke(
            timewarp,
            ["--date-added", date_added, "--force", "--uuid", UUID_DICT["palm_tree"]],
            terminal_width=TERMINAL_WIDTH,
        )
        assert result.exit_code == 0
        time.sleep(1)
    
        result = runner.invoke(
            timewarp,
            ["--inspect", "--plain", "--force", "--uuid", UUID_DICT["palm_tree"]],
            terminal_width=TERMINAL_WIDTH,
        )
        output_values = parse_inspect_output(result.output, date_added=True)
>       assert compare_inspect_output(output_values[0], expected)
E       AssertionError: assert False
E        +  where False = compare_inspect_output(InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-20 05:02:03+0000', date_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2023-01-23 04:20:34+0000'), InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-19 21:02:03-0800', date_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2022-01-01 00:00:00+0000'))

tests/test_cli_timewarp.py:1387: AssertionError
------------------------------------------------------- Captured stderr call -------------------------------------------------------
2026-07-18 13:58:04,955 - osxphotos - WARNING - parse_timewarp_output.py - 110 - compare_inspect_output: date_added does not match: 2023-01-23 04:20:34+00:00 != 2022-01-01 00:00:00+00:00
-------------------------------------------------------- Captured log call ---------------------------------------------------------
WARNING  osxphotos:parse_timewarp_output.py:110 compare_inspect_output: date_added does not match: 2023-01-23 04:20:34+00:00 != 2022-01-01 00:00:00+00:00
__________________________________________ test_date_added[2022-01-01 01:02:03-expected1] __________________________________________

photoslib = <photoscript.PhotosLibrary object at 0x1109a45a0>
suspend_capture = <tests.conftest.suspend_capture.<locals>.suspend_guard object at 0x11078d160>, date_added = '2022-01-01 01:02:03'
expected = InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-19...te_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2022-01-01 01:02:03+0000')

    @pytest.mark.timewarp
    @pytest.mark.parametrize(
        "date_added,expected",
        TEST_DATA["date_added"]["data"],
    )
    def test_date_added(
        photoslib, suspend_capture, date_added: str, expected: InspectValuesDateAdded
    ):
        """Test --date-added"""
    
        runner = CliRunner()
        result = runner.invoke(
            timewarp,
            ["--date-added", date_added, "--force", "--uuid", UUID_DICT["palm_tree"]],
            terminal_width=TERMINAL_WIDTH,
        )
        assert result.exit_code == 0
        time.sleep(1)
    
        result = runner.invoke(
            timewarp,
            ["--inspect", "--plain", "--force", "--uuid", UUID_DICT["palm_tree"]],
            terminal_width=TERMINAL_WIDTH,
        )
        output_values = parse_inspect_output(result.output, date_added=True)
>       assert compare_inspect_output(output_values[0], expected)
E       AssertionError: assert False
E        +  where False = compare_inspect_output(InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-20 05:02:03+0000', date_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2023-01-23 04:20:34+0000'), InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-19 21:02:03-0800', date_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2022-01-01 01:02:03+0000'))

tests/test_cli_timewarp.py:1387: AssertionError
------------------------------------------------------- Captured stderr call -------------------------------------------------------
2026-07-18 13:58:06,318 - osxphotos - WARNING - parse_timewarp_output.py - 110 - compare_inspect_output: date_added does not match: 2023-01-23 04:20:34+00:00 != 2022-01-01 01:02:03+00:00
-------------------------------------------------------- Captured log call ---------------------------------------------------------
WARNING  osxphotos:parse_timewarp_output.py:110 compare_inspect_output: date_added does not match: 2023-01-23 04:20:34+00:00 != 2022-01-01 01:02:03+00:00
____________________________________________________ test_date_added_from_photo ____________________________________________________

photoslib = <photoscript.PhotosLibrary object at 0x110863890>
suspend_capture = <tests.conftest.suspend_capture.<locals>.suspend_guard object at 0x11078d400>

    @pytest.mark.timewarp
    def test_date_added_from_photo(photoslib, suspend_capture):
        """Test --date-added-from-photo"""
    
        runner = CliRunner()
        result = runner.invoke(
            timewarp,
            ["--date-added-from-photo", "--force", "--uuid", UUID_DICT["palm_tree"]],
            terminal_width=TERMINAL_WIDTH,
        )
        assert result.exit_code == 0
        time.sleep(1)
    
        result = runner.invoke(
            timewarp,
            ["--inspect", "--plain", "--force", "--uuid", UUID_DICT["palm_tree"]],
            terminal_width=TERMINAL_WIDTH,
        )
        output_values = parse_inspect_output(result.output, date_added=True)
        expected = TEST_DATA["date_added_from_photo"]["expected"]
>       assert compare_inspect_output(output_values[0], expected)
E       AssertionError: assert False
E        +  where False = compare_inspect_output(InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-20 05:02:03+0000', date_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2023-01-23 04:20:34+0000'), InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-19 21:02:03-0800', date_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2023-01-19 21:02:03-0800'))

tests/test_cli_timewarp.py:1410: AssertionError
------------------------------------------------------- Captured stderr call -------------------------------------------------------
2026-07-18 13:58:07,586 - osxphotos - WARNING - parse_timewarp_output.py - 110 - compare_inspect_output: date_added does not match: 2023-01-23 04:20:34+00:00 != 2023-01-19 21:02:03-08:00
-------------------------------------------------------- Captured log call ---------------------------------------------------------
WARNING  osxphotos:parse_timewarp_output.py:110 compare_inspect_output: date_added does not match: 2023-01-23 04:20:34+00:00 != 2023-01-19 21:02:03-08:00
===================================================== short test summary info ======================================================
FAILED tests/test_cli_timewarp.py::test_date_added[2022-01-01-expected0] - AssertionError: assert False
FAILED tests/test_cli_timewarp.py::test_date_added[2022-01-01 01:02:03-expected1] - AssertionError: assert False
FAILED tests/test_cli_timewarp.py::test_date_added_from_photo - AssertionError: assert False
============================================= 3 failed, 58 passed in 124.11s (0:02:04) =============================================
```

</details>


### run_scripts test #4: OK

### run_scripts test #5: Two Failures

<details>

<summary>
Two errors.
</summary>

```
============================================================= FAILURES =============================================================
______________________________________________________ test_import_dup_check _______________________________________________________

    @pytest.mark.test_import
    def test_import_dup_check():
        """Test basic import with --dup-check"""
        say("Please click Import when prompted by Photos to import duplicate photo.")
    
        cwd = os.getcwd()
        test_image_1 = os.path.join(cwd, TEST_IMAGE_1)
        runner = CliRunner()
    
        # import first to ensure photo is in library
        result = runner.invoke(
            import_main,
            ["--verbose", test_image_1],
            terminal_width=TERMINAL_WIDTH,
        )
    
        # now import again with --dup-check
        result = runner.invoke(
            import_main,
            ["--verbose", "--dup-check", test_image_1],
            terminal_width=TERMINAL_WIDTH,
        )
        assert result.exit_code == 0
    
        import_data = parse_import_output(result.output)
        file_1 = pathlib.Path(test_image_1).name
>       uuid_1 = import_data[file_1]
E       KeyError: 'IMG_4179.jpeg'

tests/test_cli_import.py:338: KeyError
_______________________________________ test_import_exportdb_datetime_set_timezone_timezone ________________________________________

tmp_path = PosixPath('/private/var/folders/rt/1jgsw3nj6sz558zzjnwkgqrm0000gp/T/pytest-of-msp/pytest-128/test_import_exportdb_datetime_2')

    @pytest.mark.test_import
    def test_import_exportdb_datetime_set_timezone_timezone(tmp_path):
        """Test osxphotos import with --exportdb option to verify date/time/timezone is set correctly with --timezone"""
    
        # first, export an image
        runner = CliRunner()
        result = runner.invoke(
            export,
            [
                "--verbose",
                str(tmp_path),
                "--name",
                "wedding.jpg",  # has an invalid date in the library so photo.date == 1970-01-01 00:00:00+00:00
                "--library",
                TEST_EXPORT_LIBRARY,
            ],
        )
        assert result.exit_code == 0
    
        # now import that exported photo with --exportdb
        result = runner.invoke(
            import_main,
            [
                str(tmp_path),
                "--glob",
                "wedding.jpg",
                "--verbose",
                "--exportdb",
                str(tmp_path),
                "--set-timezone",
                "--force",
                "--timezone",
                "America/New_York",
            ],
            terminal_width=TERMINAL_WIDTH,
        )
        assert result.exit_code == 0
        assert "Setting metadata and location from export database" in result.output
        results = parse_import_output(result.output)
        photosdb = PhotosDB()
        photo = photosdb.query(QueryOptions(uuid=[results["wedding.jpg"]]))[0]
        actual_date = datetime.datetime(
            2019, 4, 15, 14, 40, 24, tzinfo=ZoneInfo(key="America/New_York")
        )
>       assert photo.date == actual_date
E       AssertionError: assert datetime.datetime(2019, 4, 15, 19, 40, 24, tzinfo=zoneinfo.ZoneInfo(key='America/New_York')) == datetime.datetime(2019, 4, 15, 14, 40, 24, tzinfo=zoneinfo.ZoneInfo(key='America/New_York'))
E        +  where datetime.datetime(2019, 4, 15, 19, 40, 24, tzinfo=zoneinfo.ZoneInfo(key='America/New_York')) = osxphotos.PhotoInfo(db=osxphotos.PhotosDB(dbfile='/Users/msp/Pictures/Test-15.4.1.photoslibrary/database/Photos.sqlite'), uuid='7C404308-2F14-497E-AD08-B4C54D3D5AC9', info={'_uuid': '7C404308-2F14-497E-AD08-B4C54D3D5AC9', 'modelID': None, 'masterUuid': None, 'masterFingerprint': None, 'name': '', 'lastmodifieddate_timestamp': None, 'lastmodifieddate': None, 'imageTimeZoneOffsetSeconds': -14400, 'imageTimeZoneName': 'America/New_York', 'imageDate_timestamp': 577064424, 'imageDate': datetime.datetime(2019, 4, 15, 19, 40, 24, tzinfo=zoneinfo.ZoneInfo(key='America/New_York')), 'hidden': 0, 'favorite': 1, 'originalFilename': 'wedding.jpg', 'filename': '7C404308-2F14-497E-AD08-B4C54D3D5AC9.jpeg', 'directory': '7', 'latitude': None, 'longitude': None, 'hasAdjustments': 0, 'cloudbatchpublishdate': None, 'shared': False, 'extendedDescription': 'Bride Wedding day', 'localAvailability': 1, 'remoteAvailability': 0, 'isMissing': 0, 'adjustmentUuid': None, 'adjustmentFormatID': None, 'type': 0, 'UTI': 'public.jpeg', 'UTI_original': None, 'burstUUID': None, 'burstPickType': 0, 'burst': False, 'subtype': 0, 'live_photo': False, 'screenshot': False, 'screen_recording': False, 'slow_mo': False, 'time_lapse': False, 'customRenderedValue': 0, 'hdr': False, 'depth_state': 0, 'portrait': False, 'spatial': 0, 'panorama': False, 'selfie': False, 'cloudAssetGUID': None, 'cloudLocalState': None, 'incloud': None, 'cloudLibraryState': None, 'cloudStatus': None, 'cloudAvailable': None, 'cloudMasterGUID': None, 'reverse_geolocation': None, 'placeIDs': None, 'placeNames': None, 'countryCode': None, 'momentID': 49, 'original_resource_choice': 0, 'raw_is_original': False, 'intrash': False, 'trasheddate_timestamp': None, 'trasheddate': None, 'height': 1367, 'width': 2048, 'orientation': 1, 'original_height': 1367, 'original_width': 2048, 'original_orientation': 1, 'original_filesize': 460483, 'visibility_state': 0, 'visible': True, 'saved_asset_type': 3, 'isreference': False, 'added_date': datetime.datetime(2026, 7, 18, 15, 3, 0, 827622, tzinfo=datetime.timezone(datetime.timedelta(seconds=3600), 'WEST')), 'pk': 81, 'cloudownerhashedpersonid': None, 'moment_share': None, 'imported_by_display_name': 'Photos', 'imported_by_bundle_id': 'com.apple.Photos', 'import_session': 149, 'fok_import_session': None, 'import_uuid': 'D6D552FC-2F94-44D2-A63B-38136D6BA4B3', 'has_raw': False, 'raw_data_length': None, 'UTI_raw': None, 'datastore_subtype': None, 'resource_type': None, 'raw_master_uuid': None, 'non_raw_master_uuid': None, 'alt_master_uuid': None, 'raw_info': None, 'edit_resource_id_photo': None, 'edit_resource_id_video': None, 'UTI_edited_photo': None, 'UTI_edited_video': None, 'active_library_participation_state': 0, 'library_scope_share_state': 0, 'library_scope': None, 'hasKeywords': 1, 'keywords': ['wedding'], 'hasPersons': 0, 'persons': [], 'albums': ['973ED0FD-5B5F-4CD7-A40F-4DDE73CE3FAB', 'EA8E27F6-2A49-44B0-BC77-2A2BC23C21BF'], 'hasAlbums': 1}).date

tests/test_cli_import.py:1934: AssertionError
===================================================== short test summary info ======================================================
FAILED tests/test_cli_import.py::test_import_dup_check - KeyError: 'IMG_4179.jpeg'
FAILED tests/test_cli_import.py::test_import_exportdb_datetime_set_timezone_timezone - AssertionError: assert datetime.datetime(2019, 4, 15, 19, 40, 24, tzinfo=zoneinfo.ZoneInfo(key='America/New_York')) == datetime...
============================================= 2 failed, 63 passed in 147.32s (0:02:27) =============================================
```

</details>


### run_scripts test 6: OK

### run_scripts test 7: OK 

### run_scripts test 8: OK 

### run_scripts test 9: OK

### run_scripts test 10: NOT EXECUTED

> E       AssertionError: assert False
> E        +  where False = compare_inspect_output(InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-20 05:02:03+0000', date_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2023-01-23 04:20:34+0000'), InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-19 21:02:03-0800', date_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2022-01-01 00:00:00+0000'))
> 
> tests/test_cli_timewarp.py:1387: AssertionError

@RhetTbull Is the failure maybe caused by the local Timezone of my machine? I recall talking about it with you some time back.

### Update: run_scripts test 5: OK

```
=== 65 passed in 131.82s (0:02:11) ===
```

### Update: run_scripts test 3: Getting some errors: will investigate.

<details>

<summary>
Only one test fails
</summary>


```
===================================================== FAILURES =====================================================
__________________________________ test_date_added[2022-01-01 01:02:03-expected1] __________________________________

photoslib = <photoscript.PhotosLibrary object at 0x10f378dc0>
suspend_capture = <tests.conftest.suspend_capture.<locals>.suspend_guard object at 0x10e69d160>
date_added = '2022-01-01 01:02:03'
expected = InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-19...te_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2022-01-01 01:02:03+0000')

    @pytest.mark.timewarp
    @pytest.mark.parametrize(
        "date_added,expected",
        TEST_DATA["date_added"]["data"],
    )
    def test_date_added(
        photoslib, suspend_capture, date_added: str, expected: InspectValuesDateAdded
    ):
        """Test --date-added"""
    
        runner = CliRunner()
        result = runner.invoke(
            timewarp,
            ["--date-added", date_added, "--force", "--uuid", UUID_DICT["palm_tree"]],
            terminal_width=TERMINAL_WIDTH,
        )
        assert result.exit_code == 0
        time.sleep(1)
    
        result = runner.invoke(
            timewarp,
            ["--inspect", "--plain", "--force", "--uuid", UUID_DICT["palm_tree"]],
            terminal_width=TERMINAL_WIDTH,
        )
        output_values = parse_inspect_output(result.output, date_added=True)
>       assert compare_inspect_output(output_values[0], expected)
E       AssertionError: assert False
E        +  where False = compare_inspect_output(InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-20 05:02:03+0000', date_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2022-01-01 00:00:00+0000'), InspectValues(filename='20230120_010203-0400.jpg', uuid='5285C4E2-BB1A-49DF-AEF5-246AA337ACAB', date_local='2023-01-19 21:02:03-0800', date_tz='2023-01-20 01:02:03-0400', tz_offset='-0400', tz_name='America/Anguilla', date_added='2022-01-01 01:02:03+0000'))

tests/test_cli_timewarp.py:1387: AssertionError
----------------------------------------------- Captured stderr call -----------------------------------------------
2026-07-18 20:28:04,077 - osxphotos - WARNING - parse_timewarp_output.py - 110 - compare_inspect_output: date_addeddoes not match: 2022-01-01 00:00:00+00:00 != 2022-01-01 01:02:03+00:00
------------------------------------------------ Captured log call -------------------------------------------------
WARNING  osxphotos:parse_timewarp_output.py:110 compare_inspect_output: date_added does not match: 2022-01-01 00:00:00+00:00 != 2022-01-01 01:02:03+00:00
============================================= short test summary info ==============================================
FAILED tests/test_cli_timewarp.py::test_date_added[2022-01-01 01:02:03-expected1] - AssertionError: assert False
===================================== 1 failed, 60 passed in 130.77s (0:02:10) =====================================
(venv) 
```

</details>

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -9,7 +9,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        os: [ubuntu-latest, macos-15]
+        os: [ubuntu-latest, macos-15, macos-26]
         # os: [macos-14]
         python-version: ['3.11', '3.12', '3.13', '3.14']
     steps:

--- a/requirements.txt
+++ b/requirements.txt
@@ -50,7 +50,7 @@ toml>=0.10.2,<0.12.0
 tzdata
 utitools>=0.3.0
 whenever>=0.8.3,<0.9.0
-wrapt>=1.14.1,<=2.2.2
+wrapt>=2.0.0,<3.0.0
 wurlitzer>=3.0.2,<4.0.0
 xdg-base-dirs>=6.0.0; python_version >= '3.10'
 xdg==5.1.1; python_version <= '3.9'

--- a/requirements.txt
+++ b/requirements.txt
@@ -50,7 +50,7 @@ toml>=0.10.2,<0.12.0
 tzdata
 utitools>=0.3.0
 whenever>=0.8.3,<0.9.0
-wrapt>=1.14.1,<2.0.0
+wrapt>=1.14.1,<=2.2.2
 wurlitzer>=3.0.2,<4.0.0
 xdg-base-dirs>=6.0.0; python_version >= '3.10'
 xdg==5.1.1; python_version <= '3.9'

--- a/setup.py
+++ b/setup.py
@@ -135,7 +135,7 @@ setup(
         "tzdata",
         "utitools>=0.3.0",
         "whenever>=0.8.3,<0.9.0",
-        "wrapt>=1.14.1,<2.0.0",
+        "wrapt>=1.14.1,<=2.2.2",
         "wurlitzer>=3.0.2,<4.0.0",
         "xdg-base-dirs>=6.0.0; python_version >= '3.10'",
         "xdg==5.1.1; python_version <= '3.9'",

--- a/setup.py
+++ b/setup.py
@@ -135,7 +135,7 @@ setup(
         "tzdata",
         "utitools>=0.3.0",
         "whenever>=0.8.3,<0.9.0",
-        "wrapt>=1.14.1,<=2.2.2",
+        "wrapt>=2.0.0,<3.0.0",
         "wurlitzer>=3.0.2,<4.0.0",
         "xdg-base-dirs>=6.0.0; python_version >= '3.10'",
         "xdg==5.1.1; python_version <= '3.9'",

